### ISSUE TITLE
Show ClassNames in DEV

### ISSUE THREAD

**Issue by jrodl3r** [reactGo/reactGo#876]
Anybody has a quick solution to disable classname hashing in DEV mode?

ex:
<img width="486" alt="screen shot 2017-05-12 at 2 36 42 pm" src="https://cloud.githubusercontent.com/assets/869411/26017830/6ddecd64-3720-11e7-9b10-a49076f7f4ef.png">

**Comment by psimyn** [comment]
Changing [this line](https://github.com/reactGo/reactGo/blob/master/webpack/rules/css.js#L26) should do it

```
let localIndentName = 'localIdentName=[name]__[local]___[hash:base64:5]';
if (!production) {
  localIndentName = 'localIdentName=[name]';
}
```

**Comment by jrodl3r** [comment]
@psimyn hey buddy, that didn't seem to fix it for me =\

**Comment by jrodl3r** [comment]
@psimyn any other ideas? sorry to bother =)

**Comment by psimyn** [comment]
ah, it is a typo in the property name!

should be `localIdentName` not `localIndentName` - so the current one is broken and using the default.

I think if you fix the variable name we won't even need to the conditional - it will then have the classname before the hash.

Feel free to submit MR if you can fix this, otherwise I'll update when I'm on other computer

**Comment by jrodl3r** [comment]
I will setup a PR asap. Thx!

**Comment by choonkending** [comment]
Good find @psimyn

**Comment by jrodl3r** [comment]
@psimyn unfortunately, the classNames are now appearing "readable" on PROD after this change... Any ideas?

**Comment by GGAlanSmithee** [comment]
@jrodl3r Out of interest, why do you consider it a problem that your css classes are shown? Anyway, to answer your question, it's because you are still useing `name` even in prod; 

`let localIndentName = 'localIdentName=[name]__[local]___[hash:base64:5]';`.

Remove it to something like;

`let localIndentName = 'localIdentName=[local]___[hash:base64:5]';`.

Or even; 

`let localIndentName = 'localIdentName=[hash:base64:5]';`.

**Comment by jrodl3r** [comment]
@GGAlanSmithee it adds another layer of user-visible security. For example, say some cheese-head comes along and starts inspecting your code, the scrambled class-names are pretty good sign that you have a legitly secured system, and they're more likely to keep moving along to the next potential target. Just optics more than anything for me ; ) Thx! 👍

**Comment by jrodl3r** [comment]
Fixed by #884
